### PR TITLE
Feat/cse 317 change target endpoint parameters

### PR DESCRIPTION
**Pull request by danvanhorn**
# Changes introduced

Changes the `target/` endpoint to take the target short code as a path parameter. Like this,
```
api/target/M.GHS.C2.TA
``` 

## Related issue(s):

- CSE-298
- CSE-317

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [ ] ~~Added new stories for created/modified components (if applicable)~~
- [x] Updated Postman library for created/modified routes (if applicable)
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] ~~Verified that stories were written/modified (if applicable)~~
- [ ] Verified that Postman was updated (if applicable)
- [ ] Checked out the branch and tested changes locally

### CODE REVIEW

File: src/dal/interface/index.ts [hunk 1]
```diff
@@ -233,18 +233,14 @@ export class DbClient implements IDbClient {
...
     if (this.db) {
       try {
         result = (await this.db.collection('claims').findOne({
-          subject,
-          grades,
-          claimNumber,
           'target.shortCode': targetShortCode
         })) as IClaim;
-        result.target = result.target.filter(t => t.shortCode === targetShortCode);
+        // result.target = result.target.filter(t => t.shortCode === targetShortCode);
```
Comment (puregarlic) [inline src/dal/interface/index.ts]: Is there some reason why this line is commented out and not just removed?

File: package.json [hunk 2]
```diff
@@ -113,6 +113,7 @@
     "jest-junit": "^5.1.0",
     "lint-staged": "^7.2.0",
     "nodemon": "^1.18.3",
+    "merge": ">=1.2.1",
```
Comment (puregarlic) [inline package.json]: This doesn't appear to be used anywhere, so it should probably be removed.

File: src/dal/interface/index.ts (hunk 1 above)
Comment (danvanhorn) [reply src/dal/interface/index.ts]: I can definitely not think of a good reason for that, I'll remove it.

**Review by puregarlic — APPROVED** [review]
Awesome, thanks for refactoring the export syntax so that individual functions are exported.

**Pull request merged**